README.md

```diff
@@ -1 +1,120 @@
-# oop-school-library
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[decode-morse-code\] ](#-decode-morse-code-)
```

Comment:
- [ ] Greetings! 📚 Just a quick heads-up: this project revolves around an Object-Oriented Programming (OOP) School Library. 🏫✨ Could you please tweak the header in the table of contents to align with this theme? 🔄 Let's ensure the documentation reflects the essence of our OOP-focused endeavor. 🌐 Thanks a bunch! 🙌

Reply:
Done